commands/command.go

```diff
@@ -219,6 +219,7 @@ func (c *BaseCommand) run(args []string) error {
 	}
 
 	c.TelemetryService.SetUser(user.PublicAPIKey)
+	c.TelemetryService.SetVersion(utils.CLIVersion)
```

Comment:
do we need to "set" this explicitly?  or can it just be referenced directly when setting the "version" value on `Track` calls?